model/src/channel/message/reaction.rs

```diff
@@ -1,10 +1,14 @@
 use crate::channel::ReactionType;
 use serde::{Deserialize, Serialize};
 
+/// Message reaction struct.
```

Comment:
This is obviously not a very useful doc, but I saw this on newer models so I figured that it's fine